tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmithyCodegenPlugin.java

```diff
@@ -0,0 +1,27 @@
+package com.amazonaws.util.awsclientsmithygenerator.generators;
+
+import software.amazon.smithy.build.PluginContext;
```

Comment:
nit we dont need any of the imports yet, we shouldnt check them in with this revision

Reply:
ack.removed